### PR TITLE
Updating NeuVector docs for support items

### LINKED ISSUE #4182: Update Support Limitation section for Neuvector on 2.6.5 release notes and neuvector-integration doc
Neuvector can be deployed on Hardened RKE, RKE2 and Ks3 clusters with PSP enabled and on airgap clusters. This is covered in the issue https://github.com/rancher/rancher/issues/37799.

The document change is reported to remove support limitations for Neuvector 

Support Limitations
NeuVector installation is not supported on hardened clusters.
NeuVector installation is not supported on SELinux clusters.
NeuVector installation is not supported on clusters in an air-gapped environment.

The following notes can be added for the above clusters.
```
Hardened cluster RKE - Calico
  all Neuvector components are deployable if PSP is set to true 

Hardened cluster RKE2 - Calico and k3sContainerd
  Neuvector components Controller and Enforcer deployable if psp is set true
  For Manager and Scanner needs additional config change
  kubectl patch deploy neuvector-manager-pod -n cattle-neuvector-system --patch '{"spec":{"template":{"spec":{"securityContext":{"runAsUser": 5400}}}}}' 
  kubectl patch deploy neuvector-scanner-pod -n cattle-neuvector-system --patch '{"spec":{"template":{"spec":{"securityContext":{"runAsUser": 5400}}}}}'

Hardened cluster RKE2 + RHEL8.5+ SELinux enable - Canal and k3sContainerd
  Neuvector components Controller and Enforcer deployable if psp is set true
  For Manager and Scanner needs additional config change
  kubectl patch deploy neuvector-manager-pod --patch '{"spec":{"template":{"spec":{"securityContext":{"runAsUser": 5400}}}}}'
  kubectl patch deploy neuvector-scanner-pod --patch '{"spec":{"template":{"spec":{"securityContext":{"runAsUser": 5400}}}}}'

Hardened cluster K3S 
  all Neuvector components are deployable if PSP is set to true 
  
K3S Air-Gapped cluster
  all Neuvector components are deployable as normal cluster
```

### PR DESCRIPTION
**Pull request by jtravee**
Closes #4182

- Reference: [NV docs](https://rancher.com/docs/rancher/v2.6/en/neuvector-integration/)
- 2.6.5 release notes update to be done separately.

### CODE REVIEW

File: content/rancher/v2.6/en/neuvector-integration/_index.md [hunk 1]
```diff
@@ -96,6 +96,44 @@ Below are the minimum recommended computing resources for the NeuVector chart in
...
+- All NeuVector components are deployable if PSP is set to true.
+
+{{% /tab %}}
+{{% tab "RKE2" %}}
+
+- NeuVector components Controller and Enforcer are deployable if PSP is set to true.
+
+- For Manager and Scanner components, additional configuration is required as shown below:
+
+```
```
Comment (anupama2501) [inline content/rancher/v2.6/en/neuvector-integration/_index.md:113]: From this comment https://github.com/rancher/docs/issues/4182#issuecomment-1185856886 need the updater patch command to be added as well here and under RKE2 SELinux-enabled support as well.

kubectl patch cronjob neuvector-updater-pod -n cattle-neuvector-system --patch '{"spec":{"jobTemplate":{"spec":{"template":{"spec":{"securityContext":{"runAsUser": 5400}}}}}}}'

File: content/rancher/v2.6/en/neuvector-integration/_index.md [hunk 2]
```diff
@@ -96,6 +96,44 @@ Below are the minimum recommended computing resources for the NeuVector chart in
...
+kubectl patch deploy neuvector-manager-pod -n cattle-neuvector-system --patch '{"spec":{"template":{"spec":{"securityContext":{"runAsUser": 5400}}}}}' 
+kubectl patch deploy neuvector-scanner-pod -n cattle-neuvector-system --patch '{"spec":{"template":{"spec":{"securityContext":{"runAsUser": 5400}}}}}'
+```
+
+{{% /tab %}}
+{{% /tabs %}}
+</br>
+### RKE2 SELinux-enabled Cluster Support - Calico and Canal
```
Comment (anupama2501) [inline content/rancher/v2.6/en/neuvector-integration/_index.md:121]: Apologies this should be only SELinux-enabled Cluster and not have `RKE2`

Comment (jtravee) [reply content/rancher/v2.6/en/neuvector-integration/_index.md]: Removed as advised.

File: content/rancher/v2.6/en/neuvector-integration/_index.md (hunk 1 above)
Comment (jtravee) [reply content/rancher/v2.6/en/neuvector-integration/_index.md:113]: Updated as suggested.

File: content/rancher/v2.6/en/neuvector-integration/_index.md [hunk 3]
```diff
@@ -96,6 +96,46 @@ Below are the minimum recommended computing resources for the NeuVector chart in
...
+
+- All NeuVector components are deployable if PSP is set to true.
+
+{{% /tab %}}
+{{% tab "RKE2" %}}
+
+- NeuVector components Controller and Enforcer are deployable if PSP is set to true.
+
+- For Manager and Scanner components, additional configuration is required as shown below:
+
```
Comment (anupama2501) [inline content/rancher/v2.6/en/neuvector-integration/_index.md:112]: For Manager, Scanner and `Updater` is to be needed here.

File: content/rancher/v2.6/en/neuvector-integration/_index.md [hunk 4]
```diff
@@ -96,6 +96,46 @@ Below are the minimum recommended computing resources for the NeuVector chart in
...
+{{% /tab %}}
+{{% /tabs %}}
+</br>
+### SELinux-enabled Cluster Support - Calico and Canal
+
+- NeuVector components Controller and Enforcer are deployable if PSP is set to true.
+
+- For Manager and Scanner components, additional configuration is required as shown below:
+
```
Comment (anupama2501) [inline content/rancher/v2.6/en/neuvector-integration/_index.md:127]: For Manager, Scanner and Updater is to be needed here.

**Review by selvamt94 — COMMENTED** [review]
Looks Good to me. Thanks, @anupama2501 verifying them.

File: content/rancher/v2.6/en/neuvector-integration/_index.md (hunk 3 above)
Comment (jtravee) [reply content/rancher/v2.6/en/neuvector-integration/_index.md:112]: Updated as advised.

File: content/rancher/v2.6/en/neuvector-integration/_index.md (hunk 4 above)
Comment (jtravee) [reply content/rancher/v2.6/en/neuvector-integration/_index.md:130]: Updated as advised.

File: content/rancher/v2.6/en/neuvector-integration/_index.md [hunk 5]
```diff
@@ -96,6 +96,46 @@ Below are the minimum recommended computing resources for the NeuVector chart in
...
+kubectl patch deploy neuvector-manager-pod -n cattle-neuvector-system --patch '{"spec":{"template":{"spec":{"securityContext":{"runAsUser": 5400}}}}}' 
+kubectl patch deploy neuvector-scanner-pod -n cattle-neuvector-system --patch '{"spec":{"template":{"spec":{"securityContext":{"runAsUser": 5400}}}}}'
+kubectl patch cronjob neuvector-updater-pod -n cattle-neuvector-system --patch '{"spec":{"jobTemplate":{"spec":{"template":{"spec":{"securityContext":{"runAsUser": 5400}}}}}}}'
+```
+
+{{% /tab %}}
+{{% /tabs %}}
+</br>
+### SELinux-enabled Cluster Support - Calico and Canal
+
```
[the remark this answers is not in the document]
Comment (jtravee) [reply content/rancher/v2.6/en/neuvector-integration/_index.md:124]: Updated as advised.

**Review by anupama2501 — APPROVED** [review]
LGTM

**Pull request merged**